src/Mvc/Mvc.Api.Analyzers/src/ApiDiagnosticDescriptors.cs

```diff
@@ -11,35 +11,35 @@ internal static class ApiDiagnosticDescriptors
         public static readonly DiagnosticDescriptor API1000_ActionReturnsUndocumentedStatusCode =
             new DiagnosticDescriptor(
                 "API1000",
-                "Action returns undeclared status code.",
-                "Action method returns undeclared status code '{0}'.",
```

Comment:
We should be able to keep this as is, no?

Reply:
Same for all the messages that appear below

Reply:
Yep -- fixed this. I though the M.C.A update brought in new analyzers but something might've been borked with my test setup that caused it to use the 6.0 versions of the analyzers.